docs/business.yaml

```diff
@@ -144,6 +144,56 @@ paths:
                         paymentToken: '38416'
                         status: 'PENDING'
                         submitter: 'api-key-account-3445-sandbox'
+        '400':
+          description: The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, size too large, invalid request message framing, or deceptive request routing).
```

Comment:
Add additional error responses for other potential scenarios.
400:
<img width="1419" alt="image" src="https://github.com/user-attachments/assets/29f6686f-a5ae-4ff5-97db-eae058f71bb0">

<img width="1404" alt="image" src="https://github.com/user-attachments/assets/b8a9d5ac-05e6-4df4-bd7c-cf5653974183">

401:
<img width="1407" alt="image" src="https://github.com/user-attachments/assets/a383e0d8-9fb3-4f09-86c3-51ac96b597fc">

422:
<img width="1406" alt="image" src="https://github.com/user-attachments/assets/a8753928-ad87-43b4-a22c-80bfec0526d9">